rest/method_get.go

```diff
@@ -58,6 +59,20 @@ func listGet(ctx context.Context, r *http.Request, route *RouteMatch) (status in
 		e = NewError(err)
 		return e.Code, nil, e
 	}
+	// Now we can set Total.
+	list.Total = len(list.Items)
```

Comment:
Yes, getting the total count can be arbitrary complex for the storage and it is an information that is not fomally requested and most of the time not used.

We may define a way to formally ask for this info, and thus force the computation explicitly.